### DISCUSSION TITLE
Is it possible to mirror traffic on ESXi 7 VMs ?

### DISCUSSION THREAD

**Discussion by Aster-Lin** [Security-Onion-Solutions/securityonion#12399] [Unsupported Versions]
Hi, 
I am using SecurityOnion 2.3.250 on CentOS system which installed on ESXi 7.
I would like to know if it is possible to mirror the traffic of other virtual machines on the same network segment (both internal network and external traffic).

Thank you!

**Comment by cm-ops** [comment]
Can you set up a port mirror session on your vSphere Distributed Switch?
